tests/UI/Http/Rest/Controller/Events/GetEventsControllerTest.php

```diff
@@ -29,7 +29,7 @@ public function events_should_be_present_in_elastic_search()
 
         $this->refreshIndex();
 
-        $this->get('/api/events');
+        $this->get('/api/events?limit=1');
```

Comment:
Why limit it to just one event?

Reply:
Hi @Lutacon.
I added this parameter because I wanted to cover more of `QueryController`. Do you think it should be another one (new) test?

Reply:
One is same as default value so I prefer use a different number in that case but also add an assetion for the response to match with the number of events requested

Reply:
@jorge07 default value for limit is 50 (column size). I've added tests for next page in results (empty).